components/molecules/NewsletterForm/newsletter-form.tsx

```diff
@@ -44,39 +63,57 @@ const NewsletterForm = () => {
   return (
     <>
       {isSubscribed ? (
-        <div className="flex items-center justify-center gap-3 px-4 py-6 border rounded-lg h-44 bg-light-slate-1 w-80">
+        <div className="flex items-center justify-center w-full gap-3 px-4 py-6 border rounded-lg h-44 bg-light-slate-1">
           <div className="text-2xl text-center ">
             <Image className="mx-auto" src={SaucedLogo} alt="Sauced Logo" />
             <p>You’re Subscribed!</p>
           </div>
         </div>
       ) : (
-        <div className="flex flex-col gap-3 px-4 py-6 border rounded-lg w-max">
+        <div className="flex flex-col w-full gap-3 px-4 py-6 border rounded-lg">
           <div className="w-64 space-y-1">
             <h2 className="text-lg">Subscribe to our newsletter</h2>
             <p className="text-sm text-light-slate-11">Stay up to date with the latest OpenSauced news and trends!</p>
           </div>
-          <form className="w-full" onSubmit={handleSubmit}>
+          <form
+            name="newsletter"
+            data-netlify="true"
+            data-netlify-honeypot="bot-field"
```

Comment:
Remove this as the field is not in the form

```suggestion
            data-netlify-honeypot="bot-field"
```